test/jdk/java/awt/PopupMenu/PopupMenuLocation.java

```diff
@@ -106,22 +123,36 @@ private void show(MouseEvent e) {
     }
 
     private static void openPopup(final Frame frame) throws Exception {
-        Robot robot = new Robot();
-        robot.setAutoDelay(200);
         robot.waitForIdle();
         Point pt = frame.getLocationOnScreen();
-        robot.mouseMove(pt.x + frame.getWidth() / 2, pt.y + 50);
+        int x = pt.x + frame.getWidth() / 2;
+        int y = pt.y + 50;
+        robot.mouseMove(x, y);
         robot.mousePress(InputEvent.BUTTON3_DOWN_MASK);
         robot.mouseRelease(InputEvent.BUTTON3_DOWN_MASK);
-        int x = pt.x + frame.getWidth() / 2;
-        int y = pt.y + 130;
+        robot.waitForIdle();
+         y = y+50;
```

Comment:
Another instance of the spacing issue. There seems to be more in this test, so it's probably best to fix them all.

Reply:
This line has wrong indentation. I suggest using a compact syntax:

```suggestion
         y += 50;
```
or
```suggestion
         y += OFFSET;
```
if you create the `OFFSET` constant as I suggested above.